File: 4UI/7ShareTwoStoryBoard/TwoStoryBoard/ViewController.m
//
//  ViewController.m
//  TwoStoryBoard
//
//  Created by lanouhn on 15/12/28.
//  Copyright © 2015年 李杨. All rights reserved.
//

#import "ViewController.h"
#import "SecondViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

//跳转到第二个控制器的点击事件
- (IBAction)toSecondVCActon:(UIButton *)sender {
    //找到对应的Storyboard
    UIStoryboard *storyBoard = [UIStoryboard storyboardWithName:@"TwoStoryboard" bundle:nil];
    //找到对应的视图控制器
    //storyboard入口对应的控制器
    SecondViewController *secondVC = [storyBoard instantiateInitialViewController];
    [self presentViewController:secondVC animated:YES completion:nil];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 6Demo/HybridApp/JS_Native/WebView_H5Demo/WebView_H5Demo/ViewController.swift
//
//  ViewController.swift
//  WebView_H5Demo
//
//  Created by admin on 2018/5/9.
//  Copyright © 2018年 admin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    @IBAction func toFirstVC(_ sender: UIButton) {

        if sender.tag == 200 {
            let webViewVC = storyboard?.instantiateViewController(withIdentifier: "WebViewViewController")
            navigationController?.pushViewController(webViewVC!, animated: true)
        }else if sender.tag == 201 {
            let wkWebViewVC = storyboard?.instantiateViewController(withIdentifier: "WKWebViewViewController")
            navigationController?.pushViewController(wkWebViewVC!, animated: true)
        }else {
            let js_NativeCameraVC = storyboard?.instantiateViewController(withIdentifier: "JS_NativeCameraViewController")
            navigationController?.pushViewController(js_NativeCameraVC!, animated: true)
        }
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

